src/main/resources/application.properties

```diff
@@ -28,4 +30,15 @@ refundRetry.eventOffsetSeconds=${REFUND_RETRY_EVENT_BASE_INTERVAL_SECONDS}
 refundRetry.maxAttempts=${REFUND_RETRY_EVENT_MAX_ATTEMPTS}
 
 closePaymentRetry.eventOffsetSeconds=${CLOSE_PAYMENT_RETRY_EVENT_BASE_INTERVAL_SECONDS}
-closePaymentRetry.maxAttempts=${CLOSE_PAYMENT_RETRY_EVENT_MAX_ATTEMPTS}
\ No newline at end of file
+closePaymentRetry.maxAttempts=${CLOSE_PAYMENT_RETRY_EVENT_MAX_ATTEMPTS}
+
+notificationRetry.eventOffsetSeconds=${NOTIFICATION_RETRY_EVENT_BASE_INTERVAL_SECONDS}
+notificationRetry.maxAttempts=${NOTIFICATION_RETRY_EVENT_MAX_ATTEMPTS}
+
+notificationsService.uri=${NOTIFICATIONS_SERVICE_URI}
+notificationsService.readTimeout=${NOTIFICATIONS_SERVICE_READ_TIMEOUT}
+notificationsService.connectionTimeout=${NOTIFICATIONS_SERVICE_CONNECTION_TIMEOUT}
+notificationsService.apiKey=${NOTIFICATIONS_SERVICE_API_KEY}
+
```

Comment:
Remember to add these to the README too :)

Reply:
Right! Added with this https://github.com/pagopa/pagopa-ecommerce-event-dispatcher-service/pull/50/commits/312ac2cf4a6f0d3ccab28657204f97b912364ae3